src/components/Project/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+
+import TagsList from 'Components/TagsList';
+import PropTypes from 'prop-types';
+
+import { minutesToHours } from 'date-fns/esm';
```

Comment:
Recorda el orden de los imports
```suggestion
import React, { useState, useEffect } from 'react';
import { minutesToHours } from 'date-fns/esm';
import PropTypes from 'prop-types';

import TagsList from 'Components/TagsList';

```